mobile/src/views/MyICardView.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
```

Comment:
I agree that we can clean this up on the next sprint

Reply:
Definitely. Will put more thought in organizing the structure of styling.